src/serde/mod.rs

```diff
@@ -0,0 +1,80 @@
+pub mod de;
+pub mod decimal;
+pub mod ser;
+pub mod timestamp;
+
+pub use de::{from_ion, Deserializer};
+pub use ser::{to_binary, to_pretty, to_string, Serializer};
+
+#[cfg(test)]
+#[cfg(feature = "experimental-serde")]
+mod tests {
+    use crate::serde::{from_ion, to_string};
+
+    use crate::{Decimal, Timestamp};
+    use chrono::{DateTime, FixedOffset, Utc};
+    use serde::{Deserialize, Serialize};
+    use serde_with::serde_as;
+
+    #[test]
+    fn test_struct() {
+        #[serde_as]
+        #[derive(Serialize, Deserialize)]
+        struct Test {
```

Comment:
Can you add a field that is `Optional<_>`?

Reply:
Yes, you can optional field and that will be considered as `null` in terms of Ion representation